scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerServiceAsync.java

```diff
@@ -393,4 +393,10 @@ void getUsage(String sessionId, String user, Date startDate, Date endDate,
     void removeThirdPartyCredential(String sessionId, String key, AsyncCallback<Void> asyncCallback);
 
     Request getPreciousTaskName(String sessionId, String jobId, AsyncCallback<String> callback);
+
+    /**
+     * @param sessionId d of the current session
+     * @return true if user has right to access RM portal
+     */
+    void portalAccess(String sessionId, AsyncCallback<String> callback);
```

Comment:
The signature doesn't match w/ the javadoc. Please clarify.

Reply:
ok